bitmap/iff.cpp

```diff
@@ -336,7 +336,7 @@ int bm_iff_parse_delta(CFILE *ifile, int len, iff_bitmap_header *bmheader) {
     }
 
     if (cnt == -1) {
-      if (!bmheader->w & 1)
+      if ((!bmheader->w) & 1)
```

Comment:
This may be dangerous. Need actual testing, both `!(bmheader->w & 1)` and `(!bmheader->w) & 1` may be intended by code logic.

Reply:
`((!bmheader->w) & 1)` is what the compiler does, I just make it explicit

Reply:
(again, what clang told me in the warning message)

Reply:
Yes, but we can't tell what intended logic was be. Maybe there already placed bug which clang is pointing to.

Reply:
Dropping this modification for now. Looking at the rest of the code it's more likely `!(bmheader->w & 1)` but needs more testing